gradle/verification-metadata.xml

```diff
@@ -4321,6 +4321,9 @@
          </artifact>
       </component>
       <component group="org.junit" name="junit-bom" version="5.9.0">
+         <artifact name="junit-bom-5.9.0.module">
```

Comment:
Why do we have both JUnit 5.9.0 and 5.9.1?  Lets spend some time investigating what can be done to remove multiple versions of the same library first.